frontend/src/features/recommendation/UploaderFilter.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Box, Button } from '@material-ui/core';
+import { FilterSection } from 'src/components';
+
+interface Props {
+  value: string;
+  onChange: () => void;
```

Comment:
`onDelete` may be a more relevant name, as the only possible action is to delete the existing filter.